include/iotc.h

```diff
@@ -357,133 +282,61 @@ extern iotc_state_t iotc_publish_data(iotc_context_handle_t iotc_h,
+ * @param [in] callback The {@link ::iotc_user_subscription_callback_t callback}
+ *     invoked after a message is published to the MQTT topic.
+ * @param [in] user_data (Optional) A pointer that to the callback function's
+ *     user_data parameter.
  */
 extern iotc_state_t iotc_subscribe(iotc_context_handle_t iotc_h,
                                    const char* topic, const iotc_mqtt_qos_t qos,
                                    iotc_user_subscription_callback_t* callback,
                                    void* user_data);
 
 /**
- * @brief Asynchronously disconnect from Cloud IoT Core.
+ * @brief Disconnects asynchronously from an MQTT broker.
  *
- * After disconnecting, the Device SDK invokes the <code>iotc_connect()</code>
- * callback with a disconnection status code in the state parameter.
+ * @details After disconnecting, the Device SDK passes the disconnection status
+ * code to iotc_connect(). You may reuse disconnected contexts to reconnect
+ * to Cloud IoT Core; you don't need to destroy and recreate contexts.
  *
- * @param [in] iotc_h A context handle created by invoking iotc_create_context.
- *
- * @retval IOTC_SOCKET_NO_ACTIVE_CONNECTION_ERROR If there is no connection for
- * this context.
- * @retval IOTC_STATE_OK If the disconnection request is enqueued.
- *
- * @see iotc_create_context
- * @see iotc_connect
+ * @param [in] iotc_h A {@link iotc_create_context() context handle}.
  */
 extern iotc_state_t iotc_shutdown_connection(iotc_context_handle_t iotc_h);
 
 /**
- * @brief Execute a function after a number of elapsed seconds.
- *
- * The callback method will be invoked by the Device SDK event system after a
- * specified number of seconds.
- *
- * The callback function has the following signature:
- *    <pre>
- *    typedef void ( iotc_user_task_callback_t ) ( const iotc_context_handle_t
- *      context_handle, const iotc_timed_task_handle_t, timed_task_handle, void*
- *      user_data );
- *    <pre>
- *
- * The callback parameters are:
- *   - in_context_handle: The context handle provided to
- * <code>iotc_schedule_timed_task</code>.
- *   - timed_task_handle: The handle <code>iotc_schedule_timed_task</code>
- * returns.
- *   - user_data: The optional data provided to
- * <code>iotc_schedule_timed_task</code>.
- *
- * @param [in] iotc_h A context handle created by invoking
- * <code>iotc_create_context</code>.
- * @param [in] iotc_user_task_callback_t* A function invoked after a specified
- *    amount of time.
+ * @brief Executes a function after an interval and returns a unique ID for the
+ * scheduled task. If the function returns an error, the ID is the opposite of
+ *     the numeric {@link ::iotc_state_t error code}.
+ *
+ * @param [in] iotc_h A {@link iotc_create_context() context handle}.
+ * @param [in] iotc_user_task_callback_t The function invoked after an interval.
+ *     The signature must be:
+ *     {@code typedef void() iotc_user_task_callback_t(const iotc_context_handle_t
```

Comment:
Add reference to this API doc instead of static code. The static code can get obsolete. I cannot provide here the docs reference but I mean this content: https://github.com/GoogleCloudPlatform/iot-device-sdk-embedded-c/blob/master/include/iotc_types.h#L58

Reply:
Done